### DISCUSSION TITLE
Is it expected that balance data is sampled at 60 second resolution?

### DISCUSSION THREAD

**Discussion by HyruleWarrior** [jertel/vuegraf#99] [Q&A] [answered]
It seems like balance only gets data every minute, where all other channels have 1 second resolution (if looking at data >1 hour old). Is this expected from the Emporia API? I don't see any balance data where `detailed=True`.

**Comment by jertel** [comment, marked as answer]
I think that's a realistic possibility. If you go into the mobile Emporia app you cannot click on Balance to get detailed charts, like you can with actual circuits. So it seems to be consistent.
